src/network/netdev/bond.c

```diff
@@ -371,6 +390,115 @@ int config_parse_arp_ip_target_address(const char *unit,
         return 0;
 }
 
+int config_parse_ad_actor_sys_prio(const char *unit,
+                                   const char *filename,
+                                   unsigned line,
+                                   const char *section,
+                                   unsigned section_line,
+                                   const char *lvalue,
+                                   int ltype,
+                                   const char *rvalue,
+                                   void *data,
+                                   void *userdata) {
+        Bond *b = userdata;
+        uint16_t v;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        r = safe_atou16(rvalue, &v);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r, "Failed to parse actor system priority '%s', ignoring: %m", rvalue);
+                return 0;
+        }
+
+        if (v == 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r, "Failed to parse actor system priority '%s'. Range is [1,65535], ignoring: %m", rvalue);
```

Comment:
wrong error parameter here. the fifth argument here should be `0`, not `r`, after all errors stored in `r` are already processed further up. also, `%m` should not be used here, after all there's no actual errno-style error to print here...

Reply:
Ah, sorry, I missed that.